fbw-a32nx/src/systems/fmgc/src/flightplanning/plans/BaseFlightPlan.ts

```diff
@@ -2655,6 +2657,14 @@ export abstract class BaseFlightPlan<P extends FlightPlanPerformanceData = Fligh
     );
     this.incrementVersion();
   }
+
+  getFinalApproachCourseFixIndex(): number {
```

Comment:
```suggestion
  /**
   * Finds the index of the final approach fix.
   * @returns The leg index, or -1 if not found.
   */
  getFinalApproachCourseFixIndex(): number {
```
Just to clarify what happens when it's not found.